### DISCUSSION TITLE
java.lang.IllegalStateException: You don't seem to use the Fluent API

### DISCUSSION THREAD

**Discussion by nandakishorkn** [jobrunr/jobrunr#766] [General]
JobRunr Version
6.0.0

JDK Version
1.8.0_371

Your SQL / NoSQL database
InMemoryStorageProvider

What happened?
Hitting the IllegalStateException intermittently upon invoking org.jobrunr.configuration.JobRunr.getBackgroundJobServer().

However, same API call works sometimes but not always.

How to reproduce?
   RecurringJobsResult rjobsResult = JobRunr.getBackgroundJobServer().getStorageProvider().getRecurringJobs();
   log.info("Total recurring jobs count : " + rjobsResult.size());
   

Relevant log output
2023-05-15 20:33:06,183 ERROR [main] [] i.m.r.Micronaut: Error starting Micronaut server: You don't seem to use the Fluent API. This method is only available if you are using the Fluent API to configure JobRunr
java.lang.IllegalStateException: You don't seem to use the Fluent API. This method is only available if you are using the Fluent API to configure JobRunr
        at org.jobrunr.configuration.JobRunr.getBackgroundJobServer(JobRunr.java:61)
        

When it works, I see the log output something like "Total recurring jobs count : 6" in the log file. Otherwise, I get the IllegalStateException as updated in the section 'Relevant log output'.

**Comment by rdehuyss** [comment]
Please do not cross post but follow the community guidelines that are presented to you when you create a bug.
